src/core/Akka.Cluster.Tests.MultiNode/LeaderElectionSpec.cs

```diff
@@ -0,0 +1,248 @@
+﻿using System.Collections.Immutable;
+using System.Linq;
+using Akka.Remote.TestKit;
+using Akka.TestKit;
+
+namespace Akka.Cluster.Tests.MultiNode
+{
+    public class LeaderElectionSpecConfig : MultiNodeConfig
+    {
+        private readonly RoleName _controller;
+        public RoleName Controller
+        {
+            get { return _controller; }
+        }
+
+        private readonly RoleName _first;
+        public RoleName First
+        {
+            get { return _first; }
+        }
+
+        private readonly RoleName _second;
+        public RoleName Second
+        {
+            get { return _second; }
+        }
+
+        private readonly RoleName _third;
+        public RoleName Third
+        {
+            get { return _third; }
+        }
+
+        private readonly RoleName _forth;
+        public RoleName Forth
+        {
+            get { return _forth; }
+        }
+
+        public LeaderElectionSpecConfig(bool failureDectectorPuppet)
+        {
+            _controller = Role("controller");
+            _first = Role("first");
+            _second = Role("second");
+            _third = Role("third");
+            _forth = Role("forth");
+
+            if (failureDectectorPuppet)
```

Comment:
No need for the conditional.
MultiNodeClusterSpec.ClusterConfig has an override that'll take failureDetectorPuppet